src/i-client-config.ts

```diff
@@ -165,4 +189,77 @@ export interface IClientConfig extends IBaseRequestConfig {
      */
     maxQueueSize?: number;
   };
+};
+
+export type IStorageOptions = {
```

Comment:
I think storage uniqueness is based on SDK which _should_ be fine for this use case as I don't imagine multiple instances all using same SDK key, but I wonder if we should include that in the documentation somewhere

Reply:
I have expounded upon this in `apiKey` docs

Reply:
Perfect thanks! And now that I think about it if the have the same API it's actually probably not too bad if they are messing with eachother's persistent storage anyways as they should be using same configuration

Reply:
documentation for this type missing

Reply:
fixed